js/actions/DeploymentActions.js

```diff
@@ -0,0 +1,59 @@
+var oboe = require("oboe");
```

Comment:
Ah yep, we should probably have been more transparent on this decision (we discussed it at some length). I would say immediately that Ajax is definitely not an afterthought for Oboe - Oboe is an Ajax library. 

One of the issues we ran into repeatedly in DCOS-UI was excessive memory usage. Marathon has more endpoints and potentially more data, so will run into the same problems at scale, potentially catastrophically. Properly used, Oboe will let us keep memory use to a minimum, and provide a very apparent performance boost for large datasets. 

One of our early targets is to remove the dependency on jQuery, so we've dropped in Oboe and we're using it in 'simple http' mode to replace `$.ajax`. We can then refactor our stores to make full use of Oboe's streaming features at a later date.